include/onnxruntime/core/framework/op_kernel.h

```diff
@@ -281,6 +281,8 @@ struct KernelCreateInfo {
   KernelCreateInfo(KernelCreateInfo&& other) noexcept
       : kernel_def(std::move(other.kernel_def)),
         kernel_create_func(std::move(other.kernel_create_func)) {}
+
+  KernelCreateInfo() = default;
 };
```

Comment:
Why is this needed?

Reply:
Otherwise linker could complain about missing of default constructor referred by BuildKernelCreateInfo<void>{...}
If I remember this correctly - your prototype branch has the line as well.